notebooks/notebooks/iclr2022/win-rate-vs-training-steps.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        ignored_adversaries=[],
</pre><p>In general having <code>[]</code> as a default argument is dangerous because it's a mutable object: if you change <code>ignored_adversaries</code> it'll get reflected in subsequent calls. I think it's harmless in this particular method, but I'd be inclined to change it to default to <code>None</code> just in case.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I didn't know about this! fixed</p><p> </p>